Infrastructure/DataBaseConnection.cs

```diff
@@ -0,0 +1,7 @@
+﻿
+namespace Simple_Inventory_Management_System.Infrastructure;
+
+public class DataBaseConnection
+{
+    public static string connectionString = "Server=LAPTOP-G8RBUTE9;Database=Inventory_Management_System;Trusted_Connection=True;TrustServerCertificate=True;";
```

Comment:
Also, have you tried using environment variables or some way to avoid publishing the connection string to the repo? 
```suggestion
    public static string ConnectionString= "Server=LAPTOP-G8RBUTE9;Database=Inventory_Management_System;Trusted_Connection=True;TrustServerCertificate=True;";
```